web/src/components/navbar/NavBarMobile.tsx

```diff
@@ -75,13 +81,26 @@ export const NavBarMobile = (props: Props): ReactElement => {
         }`}
       >
         <div className={`usa-logo ${props.scrolled ? "bg-white" : ""}`}>
-          {props.showSidebar ? (
-            <div className="text-bold font-body-sm truncate-long-business-names_NavBarMobile">
-              {getNavBarBusinessTitle(business, state.isAuthenticated)}
+          <div className={"display-flex flex-align-center"}>
+            <div className={props.showSidebar ? "width-15" : ""}>
+              <BusinessNjGovLogo />
             </div>
-          ) : (
-            <NavigatorLogo />
-          )}
+            <div className="margin-x-105">
```

Comment:
pebble - keeping things dry, this could be a component

Reply:
which portion are you referring to?

Reply:
Yeah, aren't we using it elsewhere?

Reply:
yeah, we do. I was thinking of a way to make a component out of it, but each case is slightly different enough where  the final component wouldn't be readable